ingredients/file/fileDirectory.go

```diff
@@ -32,18 +32,18 @@ func (f File) directory(ctx context.Context, test bool) (types.Result, error) {
 	if !ok {
 		return types.Result{
 			Succeeded: false, Failed: true,
-			Notes: changes, Changed: changes != nil,
+			Notes: notes, Changed: notes != nil,
```

Comment:
my fault but this looks incorrect, as notes will never be nil, since we declare it up above to be an empty slice